gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/src/DbDataStore.js

```diff
@@ -221,55 +250,54 @@ fluid.defaults("gpii.oauth2.dbDataStore", {
                 }
             }
         },
-        findAuthorizedClientsByGpiiTokenDataSource: {
+        findUserAuthorizedAuthorizationByIdDataSource: {
             type: "gpii.oauth2.dbDataSource",
             options: {
-                requestUrl: "/_design/views/_view/findAuthorizedClientsByGpiiToken?key=%22%gpiiToken%22&include_docs=true",
+                requestUrl: "/_design/views/_view/findUserAuthorizedAuthorizationById?key=%22%id%22",
                 termMap: {
-                    gpiiToken: "%gpiiToken"
+                    id: "%id"
                 },
                 rules: {
                     readPayload: {
-                        "": "rows"
+                        "": "rows.0.value"
                     }
                 }
             }
         },
-        findAuthByAccessTokenDataSource: {
+        findUserAuthorizedClientsByGpiiTokenDataSource: {
             type: "gpii.oauth2.dbDataSource",
             options: {
-                requestUrl: "/_design/views/_view/findAuthByAccessToken?key=%22%accessToken%22&include_docs=true",
+                requestUrl: "/_design/views/_view/findUserAuthorizedClientsByGpiiToken?key=%22%gpiiToken%22&include_docs=true",
                 termMap: {
-                    accessToken: "%accessToken"
+                    gpiiToken: "%gpiiToken"
                 },
                 rules: {
                     readPayload: {
-                        "": "rows.0"
+                        "": "rows"
                     }
                 }
             }
         },
-        findAuthDecisionByGpiiTokenAndClientIdDataSource: {
+        findAuthorizationByAccessTokenDataSource: {
             type: "gpii.oauth2.dbDataSource",
             options: {
-                requestUrl: "/_design/views/_view/findAuthDecisionByGpiiTokenAndClientId?key=[\"%gpiiToken\",\"%clientId\"]",
+                requestUrl: "/_design/views/_view/findAuthorizationByAccessToken?key=%22%accessToken%22&include_docs=true",
                 termMap: {
-                    gpiiToken: "%gpiiToken",
-                    clientId: "%clientId"
+                    accessToken: "%accessToken"
                 },
                 rules: {
                     readPayload: {
-                        "": "rows.0.value"
+                        "": "rows.0"
```

Comment:
We mostly use `"rows.0.value"` but in two instances use `"rows.0"`. I'm thinking it might be better to be consistent here.